fcl/protodunevd/detsim/protodunevd_refactored_detsim_Xe10ppm.fcl

```diff
@@ -0,0 +1,70 @@
+#include "services_refactored_pdune.fcl"
+#include "wirecell_dune.fcl"
+#include "opticaldetectormodules_dune.fcl"
+#include "CRTVD.fcl"
+#include "SIPMOpSensorSim.fcl"
+#include "WaveformDigitizerSim.fcl"
+#include "tools_dune.fcl"
+
+process_name: Detsim
+
+services:
+{
+  TFileService: { fileName: "detsim_single_protoDUNE_hist.root" }
+  TimeTracker:       @local::dune_time_tracker
+  MemoryTracker:     @local::dune_memory_tracker
+  RandomNumberGenerator: {} #ART native random number generator
+  FileCatalogMetadata:  @local::art_file_catalog_mc
+  @table::protodunevd_refactored_simulation_services
+}
+
+source:
+{
+  module_type: RootInput
+  maxEvents: 30000
+  fileNames: ["g4single_protoDUNE.root"]
+}
+
+physics:
+{
+
+  producers:
+  {
+    tpcrawdecoder:  @local::wirecell_protodunevd_mc
+    sipmAr10ppm:    @local::dunefdvd_xarapuca_ar_xe10ppm
+    sipmXe10ppm:    @local::dunefdvd_xarapuca_xe_xe10ppm
+    opdigi10ppm:    @local::standard_vddaphne_xe10ppm
```

Comment:
Jose, does this work properly without adjusting the input tags for the opdigi10ppm? The standard version should include  four labels: 2 for active volume (semi-ana model) and 2 for external volume (with library). I realize I have never created a configuration specifically for the computable graph in VD but this could be a good idea. The difference would be
`physics.producers.opdigi10ppm.InputTags:    [sipmAr10ppm, sipmXe10ppm]`

Reply:
yes, this is fine. All volumes (volTPC and volCryostat) are merged in a single handle during the IonAndScint step.